src/fu-util.c

```diff
@@ -528,7 +528,10 @@ fu_util_build_device_tree (FuUtilPrivate *priv, GNode *root, GPtrArray *devs, Fw
 static gchar *
 fu_util_get_tree_title (FuUtilPrivate *priv)
 {
-	return g_strdup (fwupd_client_get_host_product (priv->client));
+	return g_strdup_printf ("%s - %s %s",
+				fwupd_client_get_host_product (priv->client),
+				fwupd_client_get_host_vendor (priv->client),
+				fwupd_client_get_host_family (priv->client));
```

Comment:
This is now pretty verbose.

```
XPS 13 7390 - Dell Inc. XPS
│
```

Also the same thing needs to be set in `fu-tool.c` to stay consistent with `fwupdtool` and `fwupdmgr`

Reply:
Yup, I'll just drop this hunk for now.